litert/vendors/intel_openvino/dispatch/invocation_context.cc

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "litert/vendors/intel_openvino/dispatch/invocation_context.h"
+
+#include "litert/c/litert_common.h"
+#include "litert/c/litert_logging.h"
+#include "litert/c/litert_model.h"
+#include "litert/c/litert_tensor_buffer.h"
+#include "litert/c/litert_tensor_buffer_requirements.h"
+#include "litert/cc/litert_expected.h"
+#include "litert/core/util/tensor_type_util.h"
+#include "litert/vendors/c/litert_dispatch.h"
+
+litert::Expected<LiteRtDispatchInvocationContextT::Ptr>
+LiteRtDispatchInvocationContextT::Create(
+    ov::Core core, LiteRtDispatchDeviceContextT &device_context,
+    LiteRtDispatchExecutableType exec_type,
+    const LiteRtMemBuffer *exec_bytecode_buffer, const char *function_name,
+    int num_inputs, int num_outputs) {
+  const void *exec_bytecode_ptr =
+      static_cast<const uint8_t *>(exec_bytecode_buffer->base_addr) +
+      exec_bytecode_buffer->offset;
+  auto exec_bytecode_size = exec_bytecode_buffer->size;
+
+  std::string bytecode_buffer(reinterpret_cast<const char *>(exec_bytecode_ptr),
+                              exec_bytecode_size);
+  std::istringstream model_stream(bytecode_buffer);
+  if (!model_stream) {
+    return litert::Error(kLiteRtStatusErrorRuntimeFailure,
+                         "Failed to open model bytecode stream");
+  }
+  ov::CompiledModel compiled_model = core.import_model(model_stream, "NPU");
+  auto infer_request = compiled_model.create_infer_request();
+  LITERT_LOG(LITERT_INFO, "Openvino InvocationContext Initialize SUCCESS");
+  // TODO: add support for loading cached model
+  return Ptr(new LiteRtDispatchInvocationContextT(infer_request, device_context,
+                                                  num_inputs, num_outputs));
+}
+
+litert::Expected<LiteRtTensorBufferRequirements>
+LiteRtDispatchInvocationContextT::GetTensorBufferRequirements(
+    const LiteRtRankedTensorType &tensor_type) {
+  LiteRtTensorBufferType supported_tensor_buffer_types[] = {
+      kLiteRtTensorBufferTypeAhwb,
```

Comment:
Are these the order of your preference? Is there any performance difference between two?